src/js/views/patients/worklist/worklist-list.scss

```diff
@@ -73,8 +73,13 @@
   white-space: normal;
 }
 
+.worklist-list__item-name {
+  width: 27%;
+}
+
 .worklist-list__meta {
   text-overflow: inherit;
+  width: 43%;
```

Comment:
just did these two just to get them up. Sizing all adds up to 100% now so...yay?`